src/js/redux/reducers/search/searchFiltersReducer.js

```diff
@@ -14,6 +14,7 @@ import * as AwardAmountFilterFunctions from './filters/awardAmountFilterFunction
 import * as OtherFilterFunctions from './filters/OtherFilterFunctions';
 import * as ContractFilterFunctions from './filters/contractFilterFunctions';
 import * as ProgramSourceFilterFunctions from './filters/programSourceFilterFunctions';
+// import * as TimePeriodFilterFunctions from './filters/timePeriodFilterFunctions';
```

Comment:
should this be commented out?

Reply:
Yes, this function isn't necessary for current functionality.